static/app/utils/issueTypeConfig/index.tsx

```diff
@@ -108,6 +108,7 @@ const issueTypeConfig: Config = {
   [IssueCategory.HTTP_CLIENT]: httpClientConfig,
   [IssueCategory.DB_QUERY]: dbQueryConfig,
   [IssueCategory.MOBILE]: mobileConfig,
+  [IssueCategory.METRIC]: metricIssueConfig,
```

Comment:
@malwilley are you sure this is correct? The newly created `static/app/utils/issueTypeConfig/metricConfig.tsx` is unused, and here, we are using `static/app/utils/issueTypeConfig/metricIssueConfig.tsx`, which is the same thing used for `IssueCategory.METRIC_ALERT` 🤔

Reply:
You're right! Autocomplete got me. I have a PR up to fix and clean up other unused configs